depthai_sdk/src/depthai_sdk/replay.py

```diff
@@ -348,7 +348,7 @@ def _resize_frame(self, frame: np.ndarray, size: Tuple[int, int], mode: ResizeMo
 
     def _createNewFrame(self, cvFrame) -> dai.ImgFrame:
         imgFrame = dai.ImgFrame()
-        imgFrame.setData(cvFrame)
```

Comment:
@adamhosticka what was the error here, that was fixed using `.setFrame()`?

Reply:
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hostiada/Documents/Luxonis/Flash_RVC3_app/venv/lib/python3.10/site-packages/depthai_sdk/replay.py", line 287, in run
    if not self.sendFrames(cb): break
  File "/home/hostiada/Documents/Luxonis/Flash_RVC3_app/venv/lib/python3.10/site-packages/depthai_sdk/replay.py", line 306, in sendFrames
    stream.imgFrame = self._createImgFrame(stream)
  File "/home/hostiada/Documents/Luxonis/Flash_RVC3_app/venv/lib/python3.10/site-packages/depthai_sdk/replay.py", line 368, in _createImgFrame
    imgFrame = self._createNewFrame(cvFrame)
  File "/home/hostiada/Documents/Luxonis/Flash_RVC3_app/venv/lib/python3.10/site-packages/depthai_sdk/replay.py", line 351, in _createNewFrame
    imgFrame.setData(cvFrame)
RuntimeError: Unable to cast Python instance to C++ type (compile in debug mode for details)
```
when using replay with a path to a single image.But tested now it happens with mp4 as well.